auth-service/src/main/java/com/petclinic/authservice/businesslayer/UserServiceImpl.java

```diff
@@ -72,6 +72,12 @@ public List<UserDetails> findAllWithoutPage() {
         return userMapper.modelToDetailsList(userRepo.findAll());
     }
 
+
```

Comment:
Why is there no @Override? And this function does the exact same thing as the function above does ?

Reply:
ya I just notice I found the naming for findAllWithoutPage confusing so I just wrote my own with a naming I personally think fits better
I also added the Override thank you